.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## PR Type
+
+What kind of change does this PR introduce?
+
+> Please check any kind of changes that applies to this PR using `[x]`
+
+- [ ] Bug fix
+- [ ] Feature
+- [ ] Code style update (formatting, local variables)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] CI related changes
+- [ ] *..... (describe the other type)*
+
+## What is the current behavior?
+
+> Please describe the current behavior that you are modifying, or link to a relevant issue.
+
+Issue Number: *..... (put `N/A` if there are no relevant issue)*
```

Comment:
Refer to this article: https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue

Use "Close" instead of "Issue Number"